addons/mrp/__init__.py

```diff
@@ -11,12 +11,16 @@
 
 def _pre_init_mrp(cr):
     """ Allow installing MRP in databases with large stock.move table (>1M records)
-        - Creating the computed+stored field stock_move.is_done is terribly slow with the ORM and
-          leads to "Out of Memory" crashes
+        - Creating the computed+stored field stock_move.is_done and
+          stock_move.unit_factor is terribly slow with the ORM and leads to "Out of
+          Memory" crashes
     """
     cr.execute("""ALTER TABLE "stock_move" ADD COLUMN "is_done" bool;""")
     cr.execute("""UPDATE stock_move
                      SET is_done=COALESCE(state in ('done', 'cancel'), FALSE);""")
+    cr.execute("""ALTER TABLE "stock_move" ADD COLUMN "unit_factor" numeric;""")
```

Comment:
Because `unit_factor` doesn't set the `digits` attribute, now, it is a `double precision` not as `numeric`:
![image](https://user-images.githubusercontent.com/53175387/183371101-e67e5d26-5bd5-4b4c-9c30-7e291cc8cb08.png)